src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ADRController.kt

```diff
@@ -403,4 +431,18 @@ class ADRController(private val encryption: Encryption,
         val hash = objectMapper.readTree(response.body!!)["data"]["hash"].asText()
         return hash != newDatasetHash
     }
+
+    private fun getApiToFileTypeMappings(): Map<String, String> {
```

Comment:
couldnt this just be a val instead of a function?

Reply:
Moved this into app properties in fact, not sure if that is appropriate? Let me know if it should be somewhere else

Reply:
i think thats fine!